content/handbook/security/corporate/end-user-services/supported-apps/google-workspace/google-groups.md

```
@@ -0,0 +1,157 @@
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+Google groups は、メンバーに特定のリソースへのアクセス権を付与したり、メーリングリストとして使用されます。Google Groups はそれ自体がアプリではないため、Okta 経由では利用できませんが、[Google Groups ページ](https://groups.google.com/) に移動することで、所属しているすべてのグループを表示し、グループ設定を管理することができます。
+
+このページは、これまでに受けた最もよくある質問のいくつかへの解決策を提供することを目的としています。より詳細なサポートについては、[公式 Google Groups サポートページ](https://support.google.com/groups/) をご覧ください。
+
+## 目次
+
+- [グループアクセスリクエスト](#group-access-requests)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)